packages/@uppy/utils/src/emaFilter.js

```diff
@@ -0,0 +1,7 @@
+export default function emaFilter (newValue, previousFilteredValue, halfLife, dt) {
```

Comment:
Could you add some JSDoc explaining what EMA stands for and describes each arguments please?

Reply:
Done here https://github.com/transloadit/uppy/pull/4458/commits/73bcf42c8094be1fed23e1412b51d7b97a810e6f